src/lib.rs

```diff
@@ -481,7 +485,19 @@ mod test {
         assert_eq!(decoded_result.unwrap(), hash1);
     }  
 
-    #[cfg(feature = "bench")] 
+    #[test]
+    fn base64_error_on_empty() {
+        let decoded_result = ImageHash::from_base64("");
+        assert!(decoded_result.is_err());
+        /* FIXME: Figure out how to make the below run
+        match decoded_result {
+            Err(FromBase64Error::InvalidBase64Length) => assert!(true),
```

Comment:
What might also be good would be to add `pub use FromBase64Error;` to the top level so users who want to match on the error don't have to import from the `rustc_serialize` crate.